frontend/public/components/job.tsx

```diff
@@ -22,13 +21,14 @@ import {
   SectionHeading,
   Timestamp,
   navFactory,
-  pluralize,
 } from './utils';
 import { ResourceEventStream } from './events';
 import { JobModel } from '../models';
 
 const ModifyJobParallelism: KebabAction = (kind: K8sKind, obj: JobKind) => ({
-  label: 'Edit Parallelism',
+  label: 'workload~Edit parallelism',
+  // t('workload~Edit parallelism')
+  labelKey: 'workload~Edit parallelism',
```

Comment:
You shouldn't have both `label` and `labelKey`

```suggestion
  // t('workload~Edit parallelism')
  labelKey: 'workload~Edit parallelism',
```